roofit/roofitcore/inc/RooAbsCategoryLValue.h

```diff
@@ -65,17 +74,22 @@ class RooAbsCategoryLValue : public RooAbsCategory, public RooAbsLValue {
     return kTRUE; 
   }
 
-  // I/O streaming interface (machine readable)
-  virtual Bool_t readFromStream(std::istream& is, Bool_t compact, Bool_t verbose=kFALSE) ;
-  virtual void writeToStream(std::ostream& os, Bool_t compact) const ;
+  /// Not implemented.
+  virtual Bool_t readFromStream(std::istream&, Bool_t, Bool_t) { return true; }
+  /// Not implemented.
+  virtual void writeToStream(std::ostream&, Bool_t) const { }
```

Comment:
I realize it was already like this, but this pattern is evil...you can _actually_ leave the methods unimplemented if pure virtual is not an option

Reply:
👍 I could actually remove these!